crates/biome_css_semantic/src/semantic_model/specificity.rs

```diff
@@ -53,9 +53,9 @@ fn evaluate_any_pseudo_class(class: &AnyCssPseudoClass) -> Specificity {
         AnyCssPseudoClass::CssPseudoClassFunctionNth(_) => CLASS_SPECIFICITY,
         AnyCssPseudoClass::CssPseudoClassFunctionRelativeSelectorList(selector_list) => {
             if let Some(base) = selector_list
-                .name_token()
+                .name()
                 .ok()
-                .and_then(|name| evaluate_pseudo_function_selector(name.text()))
+                .and_then(|name| evaluate_pseudo_function_selector(&name.to_trimmed_text()))
```

Comment:
nit: `to_trimmed_text()` _can_ allocate a string.  I don't think it matters that much for this PR though, because `name()` consists of only 1 token, which is the case where this doesn't allocate memory. IMO, it would be better to chain another `and_then` to get the `&str` or `TokenText` of the token to prevent future changes accidentally causing this to allocate.

Reply:
Thank you!
Indeed, it's a good suggestion, I totally agree!